examples/trivial.go

```diff
@@ -7,17 +7,17 @@ package main
 
 import (
 	"context"
-	"flag"
 	"fmt"
 	logger "log"
+	"os"
 	"time"
 
 	"github.com/lightstep/lightstep-tracer-go"
 	"github.com/opentracing/opentracing-go"
 	"github.com/opentracing/opentracing-go/log"
 )
 
-var accessToken = flag.String("access_token", "", "your LightStep access token")
+var accessToken = os.Getenv("LIGHTSTEP_ACCESS_TOKEN")
```

Comment:
Suggestion: as with Gopkg.lock/Gopkg.toml, move these changes to a separate PR

Reply:
Reverted